src/Razor/test/testapps/Directory.Build.props

```diff
@@ -33,6 +33,8 @@
     </RestoreSources>
   </PropertyGroup>
 
+  <Import Project="$(SolutionRoot)..\..\eng\Versions.props" />
```

Comment:
Just double checking, this `Directory.Build.props` file doesn't get copied over during integration test run correct?

Reply:
It does in fact get copied: https://github.com/aspnet/AspNetCore-Tooling/blob/84bc53671fe926c150509c797a6af9aefde82e80/src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/ProjectDirectory.cs#L134-L140

Reply:
hmm, super curious how this doesn't break rename tests then because they'd attempt to restore with busted package versions.

Reply:
nm, @pranavkm clarified offline that we pass SolutionRoot into the integration tests.